README.md

```diff
@@ -5,44 +5,115 @@ Juicebox frontend application.
 - Mainnet: https://juicebox.money
 - Rinkeby: https://rinkeby.juicebox.money
 
-## Usage
+## Development
 
-### Dependencies
+### Installation
 
-- [Blocknative](https://www.blocknative.com) account
-- [Infura](https://infura.io) account
-- [Pinata](https://pinata.cloud) account
+1. Create a
+   [fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) of this
+   repository.
+1. Clone your fork and navigate to the root directory.
+1. Install project dependencies.
 
-1.  Clone the respository:
+   ```bash
+   yarn install
+   ```
 
-  ```bash
-  git clone https://github.com/jbx-protocol/juice-juicehouse.git
-  cd juice-juicehouse
-  ```
+1. Create a `.env` file in the root directory which mirrors the `.example.env`
+   file. Learn how to define each field in the `.env` file in [Setup](#setup).
 
-1.  Create an `.env` file in the root directory which mirrors the `.example.env` file.
+### Setup
 
-1.  Install dependencies and start the app:
+`juicebox-interface` relies on a number of services for development. Create an
+account for each of the following services:
 
-  ```
-  yarn install
-  yarn start
-  ```
+- [Infura](https://infura.io)
+- [Pinata](https://pinata.cloud)
+- [Blocknative](https://www.blocknative.com)
 
-## Web3 Providers:
+The following sections describe how to set up each service for local
+development.
 
-The frontend has two different providers that provide different levels of access to different chains:
+#### Infura
 
-- `readProvider`: used to read from contracts on network of injected provider (`.env` file points you at testnet or mainnet)
-- `signingProvider`: your personal wallet, connected to via [Blocknative](https://docs.blocknative.com/onboard).
+[Infura](https://infura.io) is the tool we use to connect to an Ethereum network
+(mainnet, or one of the testnets).
 
+Take the following steps to create an Infura project for local development:
+
+1. Select **Create New Project** to begin creating a new Infura project.
+1. Select the **Ethereum** option from the **Product** dropdown.
+1. Enter a **Name** (for example, `juicebox-local`).
+1. Select **Create** to create the project.
+
+Next, copy the following fields into your `.env` file:
+
+- **Project ID**. In the `.env` file, copy the **Project ID** into the
+  `REACT_APP_INFURA_ID` variable.
+- **Endpoint**. This is the Ethereum network that will be used for testing. If
+  you don't know which endpoint to use, select **mainnet**. In the `.env` file,
+  copy the network name into the `REACT_APP_INFURA_NETWORK` variable.
+
+#### Pinata
+
+[Pinata](https://pinata.cloud) is the tool we use br
```

Comment:
**question:**. I'm not clear on what we use Pinata for. Is it for uploading/fetching project images to/from IPFS or someting?